core/domain/activity_jobs_one_off.py

```diff
@@ -33,17 +41,48 @@ class IndexAllActivitiesJobManager(jobs.BaseMapReduceOneOffJobManager):
 
     @classmethod
     def entity_classes_to_map_over(cls):
-        return [exp_models.ExpSummaryModel,
+        return [exploration_models.ExpSummaryModel,
                 collection_models.CollectionSummaryModel]
 
     @staticmethod
     def map(item):
         if not item.deleted:
-            if isinstance(item, exp_models.ExpSummaryModel):
+            if isinstance(item, exploration_models.ExpSummaryModel):
                 search_services.index_exploration_summaries([item])
             else:
                 search_services.index_collection_summaries([item])
 
     @staticmethod
     def reduce(key, values):
         pass
+
+
+class SnapshotMetadataModelsIndexesJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that indexes all explorations and collections and compute their
```

Comment:
Wrong docstring. Please be sure to proofread / do a self-review of your PR before asking for review...

Reply:
Done.